R/plot3D.x3p.file.R

```diff
@@ -70,7 +76,7 @@ plot3D.x3p.file<-function(x3p.surf.file.info, num.x.pts=NULL, num.slices=NULL, a
   xinc<-as.numeric(head.info["x.inc"]) #should be microns already
   xaxis<-seq(from=0,to=xinc*(num.pts.per.line-1),length.out=num.x.pts)
```

Comment:
This is where the error is. If num.x.pts is NULL, seq stops. Changed num.x.pts to length(dec.col.idxs).